pyproject.toml

```diff
@@ -77,17 +77,14 @@ komodo-transpiler = "komodo.release_transpiler:main"
 [tool.pylint.messages_control]
 disable = [
     "bad-inline-option",
-    "consider-using-dict-items",
     "consider-using-sys-exit",
-    "consider-using-with",
     "deprecated-pragma",
     "duplicate-code",
     "file-ignored",
     "fixme",
     "line-too-long",
     "locally-disabled",
     "logging-not-lazy",
-    "logging-fstring-interpolation",
```

Comment:
I would keep this here as a global disable (and maybe comment here that this is not to be removed)